db/migrate/20200728124743_create_users.rb

```diff
@@ -0,0 +1,15 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    enable_extension("citext")
+
+    create_table :users do |t|
+      t.citext :email, null: false
+      t.string :full_name, null: false, default: ""
+      t.string :password_digest
```

Comment:
Should we add null false constraint here?

```suggestion
      t.string :password_digest, null: false
```